Rephrase pages to hide "block" as a UI term (#1031)
CodeYourFuture/curriculum: common-content/en/module/how-our-curriculum-works/morning-orientation-block/index.md

emoji='🎡'
time=10
[objectives]
    1="Locate the morning orientation activity in a day plan"
    2="Check learning objectives for morning orientation activity"
[build]
  render = 'never'
  list = 'local'
  publishResources = false

+++

> Early in most day plans, we use the morning orientation activity to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week).

Reviewer: ```suggestion
Early in most day plans, we use the morning orientation to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week).
```

Reviewer: I don't think you need to say activity this much. It especially becomes a bit confusing given the existence of the Activity Handbook, which doesn't contain this kind of activity. 😉 🫠

> Most activities have written down objectives. It's useful to know _why_ we're doing something, before we start doing it.

Reviewer: ```suggestion
Most sections have written down objectives. It's useful to know _why_ we're doing something, before we start doing it.
```

### Steps 👣

> 1. Find a day plan view where the morning orientation activity is used

Reviewer: ```suggestion
1. Find a day plan view where the morning orientation is used
```

> 1. Check the learning objectives on the morning orientation activity

Reviewer: ```suggestion
1. Read the learning objectives on the morning orientation
```

> 1. Volunteer to be the facilitator or timekeeper (you don't need to do this every week!)

Reviewer: ```suggestion
1. Volunteer to be the facilitator or timekeeper (you don't need to volunteer every week!)
```
